### DISCUSSION TITLE
is the system tray feature available in JS?

### DISCUSSION THREAD

**Discussion by mindplay-dk** [tauri-apps/tauri#7573] [Q&A]
Like the title says, is the [system tray](https://tauri.app/v1/guides/features/system-tray) feature exposed as a JS API?

If not, is that planned?

If I decided to use Tauri, how am I going to fare with zero Rust experience? how many features are Rust-only?

Thanks 😊

**Comment by FabianLars** [comment]
> Like the title says, is the [system tray](https://tauri.app/v1/guides/features/system-tray) feature exposed as a JS API?

Nope

> If not, is that planned?

Yes, but it's not set in stone yet.

> If I decided to use Tauri, how am I going to fare with zero Rust experience?

That really depends on your app.

> how many features are Rust-only?

Most noticably systrays and menus. can't think of any other feature that's implemented in rust that's not implemented in the [js api](https://tauri.app/v1/api/js/). That said, the api really only covers the most common (and mostly simple) features so if there's something missing that can't be implemented with built-in browser stuff then you'd go over to Rust typically.

**Reply by mindplay-dk** [reply]
Okay, thanks.

Yeah, system tray and menus was exactly what I needed for this idea.

Do you have an open issue tracking these features? I couldn't find them. I'd like to subscribe for notifications. 🙂

**Reply by FabianLars** [reply]
for menu it's here: https://github.com/tauri-apps/tauri/issues/6617. Not sure if there's one for systray too but i'd imagine both would be tackled at once anyway.
